src/coreComponents/physicsSolvers/wavePropagation/AcousticWaveEquationSEM.cpp

```diff
@@ -20,6 +20,7 @@
 #include "AcousticWaveEquationSEM.hpp"
 #include "AcousticWaveEquationSEMKernel.hpp"
 
+#include "dataRepository/KeyNames.hpp"
```

Comment:
You may not have to include this file. Do you remember which compilation error (if any) you had before including it?

Reply:
To be honest I don't even remember adding these ? I don't remember having any issue of include files this is very weird, I'll just remove them.